products/antixlinux.md

```diff
@@ -0,0 +1,60 @@
+---
+permalink: /antix
+alternate_urls:
+-   /antixlinux
+-   /antix-linux
+title: antiX
+category: os
+releasePolicyLink: https://www.antixforum.com/forums/topic/when-is-end-of-support-for-stable-antix-versions-17-19/#post-26424
+iconSlug: mxlinux
+activeSupportColumn: true
+versionCommand: cat /etc/os-release
+releaseColumn: false
+releaseDateColumn: true
+realseLabel: "__RELEASE_CYCLE__ (__CODENAME__)"
+purls:
+-   purl: pkg:os/mxlinux
+auto:
+-   distrowatch: antix
+    regex: '^Distribution Release: antiX (?P<major>\d)\.(?P<minor>\d)$'
+releases:
+-   releaseCycle: "21"
+    codename: "Grup Yorum"
+    releaseDate: 2021-10-31
+    latest: "22"
+    latestRelaseDate: 2022-10-19
+    support: 2024-06-30
+    eol: 2026-06-30
+    link: https://antixlinux.com/antix-22-released/
+-   releaseCycle: "19"
+    codename: "Marielle Franco / Hannie Schaft / Manolis Glezos / Grup Yorum"
+    releaseDate: 2019-10-17
+    latest: "19.4"
+    latestRelaseDate: 2021-05-21
+    support: 2022-09-10
+    eol: 2024-06-30
+    link: https://mxlinux.org/blog/mx-19-4-now-available/
+-   releaseCycle: "17"
+    codename: "Heather Heyer / Helen Keller"
+    releaseDate: 2017-10-24
+    latest: "17.4.1"
+    latestRelaseDate: 2019-03-28
+    support: 2020-06-05
+    eol: 2022-06-30
+    link: https://antixlinux.com/antix-17-4-1-now-out/
+-   releaseCycle: "15"
+    codename: "Killah P / Fusion / Berta Cáceres"
+    releaseDate: 2015-06-30
+    latest: "16.3"
```

Comment:
Couldn't find a source for 16 being part of 15 cycle

Reply:
@captn3m0 Both are based on Debian 8 "Jessie" and went EOL at the same time. I'm happy to break them out as separate releaseCycles if you'd like.

Reply:
Yes please, separate would be better.

Reply:
@captn3m0 See latest [commit](https://github.com/alexkoster/endoflife.date/commit/469e86b50603e6a867ad6506f8cf57757d72154c) on branch `antix` for separate 15 and 16 release cycles. Thanks for your time reviewing and please let me know if you'd like any further changes prior to merge.